app/models/theme_field.rb

```diff
@@ -281,6 +281,13 @@ def target_name
     Theme.targets.invert[target_id].to_s
   end
 
+  def export_path
```

Comment:
I think it would be nice to keep a single api for `file_path` - it might get confusing having both `export_path` and `file_path`. How about moving this logic into the lambda here: 

https://github.com/discourse/discourse/blob/49c172e2242394b4567102a4504e78d3bb0507d2/app/models/theme_field.rb#L349

That lambda is already passed a hash which includes `name` and `filename`.

Reply:
That turned out a lot better.